source/Calamari.AzureAppService/ArmClientExtensions.cs

```diff
@@ -74,6 +74,22 @@ public static async Task<Stream> GetPublishingProfileXmlWithSecrets(this ArmClie
                    };
         }
 
+        public static async Task<bool> IsScmPublishEnable(this ArmClient armClient, AzureTargetSite targetSite)
```

Comment:
Nit: IsScmPublishEnabled

If you have a green build don't worry about updating.